src/util.ts

```diff
@@ -1378,3 +1378,25 @@ function globWrapper(globPattern: string, cwd: string): Promise<boolean> {
         });
     });
 }
+
+export function createCombinedCancellationToken(...tokens: (vscode.CancellationToken | undefined)[]): vscode.CancellationToken {
```

Comment:
recommend add a comment describing the lifetime of the combined token, this does more than create a combined cancellation token

Reply:
Updated and improved the lifetime handling and disposing.